presto-main/src/main/java/io/prestosql/execution/CommentTask.java

```diff
@@ -58,6 +61,23 @@ public String getName()
 
             metadata.setTableComment(session, tableHandle.get(), statement.getComment());
         }
+        else if (statement.getType() == Comment.Type.COLUMN) {
+            QualifiedObjectName tableName = createQualifiedObjectName(session, statement, statement.getName().getPrefix().get());
+            Optional<TableHandle> tableHandle = metadata.getTableHandle(session, tableName);
```

Comment:
This needs to check if the name has a single part. E.g., if the user enters `COMMENT ON foo IS ...`